src/components/Buildbaron/BuildBaronTable.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import styled from "@emotion/styled/macro";
+import Badge from "@leafygreen-ui/badge";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Table } from "antd";
+import { StyledLink } from "components/styles";
+import { BuildBaron, JiraTicket } from "gql/generated/types";
+import { getDateCopy } from "utils/string";
+
+const { blue, gray } = uiColors;
+
+const columns = [
+  {
+    title: "Related tickets from JIRA",
+    render: (text: string, { key, fields }: JiraTicket): JSX.Element => (
+      <div>
+        <JiraTicketRow jiraKey={key} fields={fields} />
+      </div>
+    ),
+  },
+];
+
+export const BuildBaronTable: React.FC<{
+  eventData: BuildBaron;
+}> = ({ eventData }) => {
+  const searchReturnInfo = eventData?.searchReturnInfo;
+  const jiraIssues = searchReturnInfo?.issues;
+
+  return (
+    <Table
+      data-test-id="build-baron-table"
+      dataSource={jiraIssues}
+      rowKey={({ key }) => key}
+      columns={columns}
+      pagination={false}
+    />
+  );
+};
+
+const JiraTicketRow: React.FC<{
+  jiraKey: String;
+  fields;
+}> = ({ jiraKey, fields }) => {
+  const url = `https://jira.mongodb.org/browse/${jiraKey}`;
```

Comment:
This URL should be queried from the db since it is configurable

Reply:
https://jira.mongodb.org/browse/EVG-12988